app/src/main/java/edu/uc/reedws/musiclink/MainActivity.kt

```diff
@@ -71,4 +71,20 @@ class MainActivity() : AppCompatActivity() {
 
         addPlaylistDialogBuilder.show()
     }
+
+    //This function was intended to be an example of how you could implement the Share Button to share data to other apps but I can't get it running myself, guess I'm too out of my element on this one.
+    //If it could recognise the Share Button or look for the playlist as an array it could be useful.
+    /*
+    private fun shareButton.setOnClickListener(){
+
+        val message: String = UserMessage.text.toString()
+
+        val intent = Intent()
+        intent.action = Intent.ACTION_SEND
+        intent.putExtra(Intent.EXTRA_TEXT, message)
+        intent.type = "text/plain"
+
+        startActivity(Intent.createChooser(intent, "Share to : "))
+    }
```

Comment:
Committing and pushing commented-out code is a version control no-no, because version control offers us a better option.
- If you want to see what used to be there, you can go to history.
- If you have an experimental or incomplete feature, it should be in a separate branch.

What, specifically, did not work when you tried this out?

Reply:
shareButton, the UI action button wasn't attached to any function yet so I was trying to get it working as a simple copy between applications feature. shareButton wasn't recognized as a class or object in the Main_Activity.xml.
Since I wouldn't expect any of my features to actually be implemented this branch is already an experiment in action.